src/clj/ataru/person_service/birth_date_converter.clj

```diff
@@ -6,3 +6,10 @@
   {:post [(not= % "--")]} ;; When no match for finnish date, this would result in "--"
   (let [[_ day month year] (re-find finnish-date-regex finnish-format-date)]
     (str year "-" month "-" day)))
+
+(def yyyy-mm-dd-regex #"(\d{4})\-(\d{2})\-(\d{2})")
+
+(defn convert-to-finnish-format [yyyy-mm-dd-date]
+  {:post [(not= % "..")]}
+  (let [[_ year month day] (re-find yyyy-mm-dd-regex yyyy-mm-dd-date)]
```

Comment:
`(.format (finnish-date) (.parse (iso-date) yyyy-mm-dd-date))` lämmittäis mieltä.

Reply:
^